src/sonic_ax_impl/__main__.py

```diff
@@ -63,14 +63,14 @@ def logging_level_to_swss_level(log_level):
     return log_level_swsscommon
 
 
-if __name__ == "__main__":
+if __name__ == "__main__": # pragma: no cover
```

Comment:
Suggest not to add permanent skipper. I could force merge if this PR fails on coverage (override PR checker and merge).

Reply:
Fixed, please help force merge, all code in __main__.py are not covered by UT.